igvc_gazebo/launch/qualification.launch

```diff
@@ -28,18 +28,28 @@
     <arg name="debug" value="false"/>
   </include>
 
-  <!-- needed for rviz -->
-  <param name="robot_description" command="cat $(find igvc_description)/urdf/jessi.urdf" />
-  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
-
   <!-- convert laser scan to point cloud for LIDAR -->
-  <node name="scan_to_pointcloud" pkg="igvc_gazebo" type="scan_to_pointcloud" output="screen" >
+  <!--<node name="scan_to_pointcloud" pkg="igvc_gazebo" type="scan_to_pointcloud" output="screen" >
```

Comment:
why do we not want a pointcloud of the scan?

Reply:
The velodyne node automatically publishes to pcl::PointCloud so we done need the conversion node